lib/fog/identity/proxmox/models/pools.rb

```diff
@@ -29,7 +29,7 @@ class Pools < Fog::Collection
         def all
           load service.list_pools
         end
-   
+
         def get(id)
           all.find { |pool| pool.identity === id }
```

Comment:
@tristanrobert I assume the CaseEquality-operator is needed in these instructions?
Therefore I disabled the `Style/CaseEquality`-cop in 242da7df19e0eb985915f038cc17d94dacb52d1b